apps/webapp/src/app/meeting-add/meeting-edit.tsx

```diff
@@ -23,10 +23,11 @@ function subscribeToMeetingDocument(db: Firestore, meetingId: string, setMeeting
         id: meetingId,
         slots: normalizeSlots(meeting.slots)
       }));
+      document.title = meeting?.title ?? 'Cebulendarz';
```

Comment:
a to ciekawe, faktycznie ktoś powinien czyścić. może na routing resetowanie?

Reply:
Proponuję zmianę tytułu per odpowiedni widok jako rezultat zmiany meeting.title

Reply:
póki co redux no-no :)

Reply:
A to co jest teraz, co o tym myślisz, czyli effect, który wywołuje się jako rezultat zmiany meeting.title?

Reply:
w sumie to nawet reducera nie potrzebujesz ;)